src/scikit_package/app.py

```diff
@@ -60,36 +64,21 @@ def setup_subparsers(parser):
     # Create "create" subparser
     parser_create = parser.add_parser("create", help="Create a new package")
 
-    # Add subcommands under "create" for different
-    sub_create = parser_create.add_subparsers(
-        dest="package_type", required=True
-    )
+    # Add subcommands under "create"
+    sub_create = parser_create.add_subparsers(dest="subcommand", required=True)
 
-    # "workspace" subcommand
-    parser_create_workspace = sub_create.add_parser(
-        "workspace", help="Create a workspace package"
-    )
-    parser_create_workspace.set_defaults(func=create, package_type="workspace")
-
-    # "system" subcommand
-    parser_create_system = sub_create.add_parser(
-        "system", help="Create a system package"
-    )
-    parser_create_system.set_defaults(func=create, package_type="system")
+    # Define all "create" subcommands
+    create_subcommands = [
```

Comment:
Since we have multiple entry points, a good idea to refactor..

Test result:

<img width="846" alt="Screenshot 2025-05-27 at 10 51 59 AM" src="https://github.com/user-attachments/assets/d8b3ac56-fbd4-4ee0-b2aa-ce2dab41283b" />